modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,79 @@
+
+* Before this update, the attempt to connect to Azure DevOps with Argo CD would result in an error due to the deprecation of the `rsa-ssh` host key algorithm by the Azure DevOps Repository service. This update fixes the issue by providing support for the `rsa-ssh` host key algorithms during the communication process between Argo CD and Azure DevOps Repository service. link:https://issues.redhat.com/browse/GITOPS-4543[GITOPS-4543]
+
+* Before this update, {gitops-shortname} console plugin workloads did not schedule on infrastructure nodes when the `runOnInfra` field was enabled in the `GitOpsService` custom resource (CR). This update fixes the issue by adding the infrastructure `node-selector` on the {gitops-shortname} console plugin workloads. This enables users to configure custom node-selectors and tolerations on the `gitops-console` pod. As a result, when the `runOnInfra` field is enabled, the {gitops-shortname} console plugin pod is placed on the infrastructure nodes similarly to the other default workloads. link:https://issues.redhat.com/browse/GITOPS-4496[GITOPS-4496]
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users were unable to include context for notifications in the `NotificationsConfiguration` custom resource (CR). With this update, users can now add context for notifications by using the `context` field in the `NotificationsConfiguration` CR. This field can be also used to establish shared context across all templates defined in the `NotificationsConfiguration` CR. link:https://issues.redhat.com/browse/GITOPS-4303[GITOPS-4303]
++
+Example of  `context` field in the `NotificationsConfiguration` CR
++
+[source,yaml]
+----
+spec:
+  context:
+    key1: value1
+    key2: value2
+  templates:
+    template.app-created: |-
+      email:
+        subject: Application {{.app.metadata.name}} has been created.
+      message: Application {{.app.metadata.name}} has been created.
+      teams:
+        title: Application {{.app.metadata.name}} has been created.
+    template.app-deleted: |-
+     ...
+     ...
+----
+
+* Before this update, users accessing the ROSA cluster after hibernation were unable to log in to the Argo CD web console due to an error indicating an invalid redirect URI in the Dex configuration. With this update, users can now log in to the Argo CD web console without facing any errors when the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```

Comment:
I would change the definite article to an indefinite one here:
```suggestion
* Before this update, users accessing a ROSA cluster after hibernation were unable to log in to the Argo CD web console due to an error indicating an invalid redirect URI in the Dex configuration. With this update, users can now log in to the Argo CD web console without facing any errors when the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```

Reply:
Fixed.